index.js

```diff
@@ -274,10 +281,10 @@
     //
     // Returns an Object {n, s, e, w, nw, sw, ne, se}
     function getScreenBBox() {
-      var targetel   = target || d3.event.target;
+      var targetel   = target || d3.event.target
 
-      while ('undefined' === typeof targetel.getScreenCTM && 'undefined' === targetel.parentNode) {
-          targetel = targetel.parentNode;
+      while (targetel.getScreenCTM == null && targetel.parentNode == null) {
```

Comment:
It'd be interesting to see if there's much of a speed difference between the two versions.  This one certainly looks better. 😃